SnowRemovalDB/1 tables.sql

```diff
@@ -0,0 +1,23 @@
+use SnowRemovalDB
+go
+
+drop table if exists Client
+go
+
+create table dbo.Client(
+    ClientId int not null identity primary key,
+    FirstName varchar(30) not null constraint ck_Client_FirstName_cannot_be_blank check(FirstName <> ''),
+    LastName varchar(30) not null constraint ck_Client_LastName_cannot_be_blank check(LastName <> ''),
+    Address varchar(50) not null constraint ck_Client_Address_cannot_be_blank check(Address <> ''),
+    City varchar(25) not null constraint ck_Client_City_cannot_be_blank check(City <> ''),
+    Province varchar(25) not null constraint ck_Client_Province_cannot_be_blank check(Province <> ''),
```

Comment:
add a constraint that does not allow Canada's 3 territories.

Reply:
Are these 3 provinces? There is no column for territories, so wasn't sure how you wanted a constraint on this.

Reply:
I added as a constraint on provinces but don't know if that is correct.

Reply:
they fall under provinces...constraint is perfect